pkgs/development/compilers/lingua-franca/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, pkgs, stdenv, fetchFromGitHub, jdk11_headless, makeWrapper }:
+let
+  version = "0.1.0";
+in
+stdenv.mkDerivation {
+  name = "lfc";
+  version = version;
+
+  src = fetchTarball {
+    url = "https://github.com/lf-lang/lingua-franca/releases/download/nightly/lfc_nightly_20211103-050218.tar.gz";
```

Comment:
Nightly seems to be a moving tag which is bad because when it is updated you can no longer download the archives. Is there an alternative way to get the archive even when the tag is moved?